examples/geojson/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+// NOTE: To use this example standalone (e.g. outside of deck.gl repo)
+// delete the local development overrides at the bottom of this file
+
+// avoid destructuring for older Node version support
+const resolve = require('path').resolve;
+const webpack = require('webpack');
+
+const config = {
+  entry: {
+    app: resolve('./src/app.js')
+  },
+
+  devtool: 'source-map',
+
+  module: {
+    rules: [{
+      // Compile ES2015 using buble
+      test: /\.js$/,
+      loader: 'babel-loader',
+      include: [resolve('.')],
+      exclude: [/node_modules/]
+    }]
+  },
+
+  resolve: {
+    alias: {
+      // From mapbox-gl-js README. Required for non-browserify bundlers (e.g. webpack):
+      'mapbox-gl$': resolve('./node_modules/mapbox-gl/dist/mapbox-gl.js')
+    }
+  },
+
+  // Optional: Enables reading mapbox token from environment variable
+  plugins: [
+    new webpack.EnvironmentPlugin(['MAPBOX_ACCESS_TOKEN', 'MapboxAccessToken'])
```

Comment:
Should we standardize on one of these? Seems weird to support both. I think the capitalized version was the original react-map-gl version.